conf/messages

```diff
@@ -46,3 +47,7 @@ transferComplete.progress=8
 
 progress.totalSteps=8
 judgmentProgress.totalSteps=3
+
+notification.title=Important
```

Comment:
I'd make these a bit more specific than just `notification.`. Something like:

- `notification.savedProgress.title`
- `notification.savedProgress.heading`
- `notification.savedProgress.metadataInfo`

Potentially there could be other sorts of notifications in the future.

Reply:
Yeah, that makes sense. I have renamed it, thanks.